Makefile

```diff
@@ -22,7 +22,7 @@ DBMAINT=dbutil.py
 
 venv:
 	@echo install venv for the development environment
-	python3 -m venv venv
+	python -m venv venv
```

Comment:
It needs to be `python3`, because on many systems there is no `python` executable. Please change back.

Reply:
@sbarber2  - can you change this back?